server/src/controllers/christmas-tree.es6

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const request = require('request');
+
+const vcapConstants = require('../vcap-constants.es6');
+const christmasTreeRegulations = require('../models/forest-regulations.es6');
+const species = require('../models/species.es6');
+const forestSpecies = require('../models/forest-species.es6');
+const speciesNotes = require('../models/species-notes.es6');
+const forestLocations = require('../models/forest-locations.es6');
+
+const christmasTree = {};
+
+const translateRegulationsFromDatabaseToClient = input => {
+  return {
+    forest: {
+      id: input.id,
+      forestName: input.forestName,
+      description: input.description,
```

Comment:
It's not clear that this is being used on the front end, should we remove?

Reply:
being used in sprint-2